karma.conf.ci.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+var customLaunchers = {
```

Comment:
is this something you're going to want to keep in multiple places?  it may be a little overkilll, but maybe put this list in npm somewhere so you dont have to think about/copy+paste it